omnibus/files/private-chef-cookbooks/private-chef/libraries/private_chef.rb

```diff
@@ -89,6 +89,21 @@ module PrivateChef
   backup Mash.new
   backup["strategy"] = "tar"
 
+  #
```

Comment:
This comment might be more useful in the attributes file, that is where most of the other similar comments are.